modules/mail/assign.js

```diff
@@ -210,4 +213,8 @@ if (constants.canSendEmail) {
   }
 }
 
+function determineCalendarLink (task) {
```

Comment:
@Faithfinder, why not a modern approach?

```suggestion
const determineCalendarLink = (task) => {
```

Reply:
Both approaches are used around the codebase. I don't really have a preference, though I wouldn't say that function declarations aren't modern. There are hoisting differences, and using declaration style allows you to not think about whether the order of lines is acceptable in every particular case.